addons/project/models/project.py

```diff
@@ -167,6 +171,11 @@ def _inverse_is_favorite(self):
     def _get_default_favorite_user_ids(self):
         return [(6, 0, [self.env.uid])]
 
+    @api.model
+    def _read_group_stage_ids(self, stages, domain, order):
+        stage_ids = self.env['project.project.stage']._search([], order=order, access_rights_uid=SUPERUSER_ID)
+        return self.env['project.project.stage'].browse(stage_ids)
```

Comment:
```suggestion
    @api.model
    def _read_group_stage_ids(self, stages, domain, order):
        return self.env['project.project.stage'].with_user(SUPERUSER_ID).search([], order=order)
```
If SUPERUSER_ID is still needed (I don't get why this is needed).

Reply:
Indeed not needed since group.base_user has read rights on project.project.stage